Exercise_8/css/partials/sub-navigation/_big-screen.scss

```diff
@@ -0,0 +1,23 @@
+@import '../variables';
+
+/*SUB-NAVIGATION*/
+
+@media only screen and (min-width: $big-screen_min-width) {
+  .sub-navigation {
+    border-bottom: 0.1rem solid lightgrey;
+    display: flex;
+    justify-content: center;
+    margin: 8rem 0;
+
+    &__item {
+      font-size: 1.2rem;
+      padding: 1rem 2rem;
+      text-decoration: none;
+
+      &--highlighted {
+        @extend .sub-navigation__item;
```

Comment:
Please, remove `@extend` and add this modificator to HTML element instead.
`class="sub-navigation__item sub-navigation__item--highlighted"`